zuul.d/content_provider.yaml

```diff
@@ -27,6 +27,7 @@
       - openstack-k8s-operators/dataplane-operator
       - openstack-k8s-operators/placement-operator
       - openstack-k8s-operators/openstack-baremetal-operator
+      - openstack-k8s-operators/ovn-operator
```

Comment:
we will need to ensure https://github.com/openstack-k8s-operators/ci-framework/pull/379 also lists that new repository.

cc @raukadah